Resources/public/apps/dingEditors/image-picker.js

```diff
@@ -81,7 +84,9 @@ angular.module('toolsModule').directive('imagePicker', [
           })
 
           if (!notAllSuccess) {
-            scope.close()
+            console.log('success')
+            scope.step = 'focal-point-pick'
+            // scope.close()
```

Comment:
Keeping this for a rainy day? ;)

Reply:
The rainiest of days..

![https://media.giphy.com/media/141wuJxikpgHYY/giphy.gif](https://media.giphy.com/media/141wuJxikpgHYY/giphy.gif)